src/wing/CCPACSWingCell.cpp

```diff
@@ -189,7 +188,7 @@ bool CCPACSWingCell::IsInside(double eta, double xsi) const
     }
 }
 
-void CCPACSWingCell::ReadCPACS(TixiDocumentHandle tixiHandle, const std::string &cellXPath)
+void CCPACSWingCell::ReadCPACS(const TixiDocumentHandle& tixiHandle, const std::string &cellXPath)
```

Comment:
I understand your point. You can leave it like this.

Still, we should not mix too many different things together. This makes reviewing code changes really hard.